File: JavaCode/sg ma'am/PrimeWithinRange.java
public class PrimeWithinRange
{
    public static int prime(int n)
    {
        int count=0, i=1;
        for(    ; i<=n; i++)
        {
            if(n%i==0)
                count++;
        }
        if(count==2)
            return n;
        else
            return 0;
    }
    public static void main(String args[])
    {
        System.out.println("Prime numbers from 1 to 100 are");
        for(int i=1; i<=100; i++)
        {
            if(prime(i)!=0)
                System.out.println(prime(i));
        }
    }
}

File: CppCode/practice/prime_within_range.cpp
// prime_within_range.cpp
// prime number within a range in c++

#include<iostream>

using namespace std;

int main()
{
    int lv, uv, i, j;
    cout<<"Enter the lower limit: "<<endl;
    cin>>lv;
    cout<<"Enter the upper limit: "<<endl;
    cin>>uv;
    cout<<"Prime numbers from "<<lv<<" to "<<uv<<" are:"<<endl;
    for(i=lv; i<=uv; i++)
    {
        for(j=2; j<i; j++)
        {
            if(i%j==0)
                break;
            else
                continue;
        }
        if(i==j)
            cout<<i<<endl;
    }
    return 0;
}